#!/usr/bin/env python
PACKAGE = "mantis_controller_id"

from dynamic_reconfigure.parameter_generator_catkin import *
import math

gen = ParameterGenerator()

gen.add("p", double_t, 0, "Proportional gain for attitude angular control", 4.5, 0.0, None)
gen.add("yaw_w", double_t, 0, "Yaw gain weighting to allow for faster roll/pitch dynamics", 0.6, 0.0, 1.0)

exit(gen.generate(PACKAGE, "controller_id_node", "AngleGains"))
